internal/claudehooks/claudehooks.go

```
@@ -116,7 +125,50 @@ func isPraxisHookCommand(cmd, args string) bool {
+// Never resolve symlinks here: that is exactly what pins a hook to one version.
+func BinaryPath() (string, error) {
+	exe, err := execPathForTest()
+	if err != nil {
+		return "", err
+	}
+	if abs, aErr := filepath.Abs(exe); aErr == nil {
+		exe = abs
+	}
+	stable, lErr := exec.LookPath("praxis")
+	if lErr != nil {
+		return exe, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 145) but it returns nil

(nilerr)